src/components/EmptyState/index.js

```diff
@@ -9,7 +9,9 @@ export class EmptyState extends React.Component {
     description: PropTypes.string,
     buttonText: PropTypes.string,
     buttonOnClick: PropTypes.func,
-    buttonUrl: PropTypes.string
+    buttonUrl: PropTypes.string,
+    className: PropTypes.string
+
```

Comment:
Unneeded extra line here
```suggestion
```

Reply:
Ehh, the UI for this code suggestion is confusing. I just mean to suggest you should remove the blank line on line 14.